### ISSUE TITLE
Install Extension task fails due to invalid arguments passed to tfx

### ISSUE THREAD

**Issue by jmezach** [Microsoft/vsts-extension-build-release-tasks#89]
We're experiencing an issue with the Install Extension task which fails with the following error:

```
2018-09-12T08:28:58.9164130Z [command]/home/vsts/work/_tools/node_modules/.bin/tfx extension install --service-url https://marketplace.visualstudio.com/ --auth-type pat --token *** --publisher augurk --extension-id augurk-preview --accounts augurk
2018-09-12T08:28:59.2591730Z TFS Cross Platform Command Line Interface v0.6.1
2018-09-12T08:28:59.2605929Z Copyright Microsoft Corporation
2018-09-12T08:28:59.2618409Z error: Error: Installing extensions to multiple accounts no longer supported. Please use the following syntax to install an extension to an account/collection:
2018-09-12T08:28:59.2634573Z error: tfx extension install --service-url <account/collection url> --token <pat> --publisher <publisher> --extension-id <extension id>
2018-09-12T08:28:59.2741108Z ##[error]Failed
```

Seems like the parameters for the install command have changed, but this extension has not been updated to reflect that change. Or am I missing something here?

**Comment by jessehouwing** [comment]
Sure looks like it. Will have to dig into his. For now I suppose rolling back to an older version of tfx should work for you.

**Comment by jmezach** [comment]
Is there a way to specify the version of tfx to use? I'm using the tasks in my release pipeline, so we don't really have a package.json where we can specify the tfx version.

**Comment by jessehouwing** [comment]
You can run a npm install task before the install task runs. The tasks
search in a number of common locations before deciding to install a copy as
part of execution.

I don't know the list of paths from the top of my head, but if you check
common.ts you'll be able to see which paths are probed.

**Comment by jmezach** [comment]
Hmm, that's unfortunate. They should have made it a breaking change ;). Anyway, I'll try to force an older version and see how that goes.

**Comment by jessehouwing** [comment]
Install extension requires additional permissions on the Personal Access Token.

**Comment by jessehouwing** [comment]
I've pinned the extension to the latest 0.5 version of TFX and will need to update the task to ask for account urls to install into. Looks like the same Personal Access Token can still be used.

**Comment by jmezach** [comment]
Awesome, that seems to work now. I missed some permission on my personal access token to allow us to install the extension, but I fixed that now. So it looks like we are unblocked now.

**Comment by jessehouwing** [comment]
I'm leaving the item open until I fixed the task.

**Comment by markvriens** [comment]
I also ran to this issue. Now we have insert with NPM the latest version of Tfx-cli 0.6.3 Because a categorie breaking change has happen. Now we had to fix a problem with the install.. Because "Installing extensions to multiple accounts no longer supported." is a problem.
Only with the following command we can installed the extension:
tfx extension install --service-url https://<account>.visualstudio.com/ --auth-type pat --token <token> -publisher <publisher> --extension-id <extention-id>

**without --accounts**

Hopefully this helps to a fix
